### PR TITLE
Homework day8 colections

### PR DESCRIPTION
**Pull request by kovtunroman**

### CODE REVIEW

**Review by siarheibalashou — DISMISSED** [review]

File: src/homework/day8/collections/WaterCollectionTraining.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+                new Water("Прозрачная", "Нет"),
+                new Water("Прозрачная", "Нет"),
+                new Water("Мутная", "Аммиачный"));
+
+        printWaterForEachThrough(water, PrinterSymbol.DASH);
+    }
+
+    public static <T> void printWaterForEachThrough(List<T> inputList, PrinterSymbol printerSymbol) {
+        for (T element : inputList) {
+            System.out.println(((Water) element).getColor() + printerSymbol.getValue() + ((Water) element).getSmell());
```
Comment (siarheibalashou) [inline src/homework/day8/collections/WaterCollectionTraining.java]: maybe here need to print  Color + Water object? like (Water) element.getColor() + Water element

**Review by siarheibalashou — COMMENTED** [review]
pretty good

File: src/homework/day8/collections/WaterCollectionTraining.java (hunk 1 above)
Comment (kovtunroman) [reply src/homework/day8/collections/WaterCollectionTraining.java]: fixed

**Review by kovtunroman — COMMENTED** [review]
fixes

**Review by siarheibalashou — APPROVED** [review]

**Pull request merged**